styles/components-styles/badge.css

```diff
@@ -45,20 +45,20 @@
   height: 3.5rem;
```

Comment:
setting height is not a best practice, and you are unsetting in the other forms of txt-badge, 
any reason you need to set here a fixed height?